src/python/WMCore/WebTools/RESTFormatter.py

```diff
@@ -26,7 +28,21 @@ def __init__(self, config):
 
         TemplatedPage.__init__(self, config)
 
+    def genstreamer(self, data):
+        yield "["
+        firstItem  = True
+        for item in data:
+            if not firstItem:
+                yield ","
```

Comment:
Is this going to be generally safe? Some JS stuff has a problem with trailing commas as well.